scripts/install/opt/seagate/cortx/motr/libexec/m0_ha_sim

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 . /opt/seagate/cortx/motr/common/m0_sns_utils_common.sh
 . /opt/seagate/cortx/motr/common/m0_dix_utils_common.sh
+. /opt/seagate/cortx/motr/common/m0_spiel_sns_utils_common.sh
```

Comment:
This file needs to installed during deployment

Reply:
Resolved.